app/assets/stylesheets/welcome.scss

```diff
@@ -1,3 +1,38 @@
-// Place all the styles related to the welcome controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+
+h1 {
+  font-size: 70px;
+}
+
+h2 {
+ color: #dc6312;
```

Comment:
Color literals like `#dc6312` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 1 spaces